src/fides/api/ops/analytics.py

```diff
@@ -41,14 +40,23 @@ async def send_analytics_event(event: AnalyticsEvent) -> None:
     if CONFIG.user.analytics_opt_out:
         return
     try:
-        await analytics_client._AnalyticsClient__send(  # pylint: disable=protected-access
-            event
+        await analytics_client.send_async(event)
+    except AnalyticsError as err:
+        log.warning("Error sending analytics event: {err}")
+    else:
+        log.info(
+            "Analytics event sent: {event.event} with client id: {analytics_client.client_id}"
         )
+
+
+async def send_registration(registration: UserRegistration) -> None:
+    if CONFIG.user.analytics_opt_out:
+        return
```

Comment:
OK, so if analytics opt out is set, we won't send either analytics events or registrations. I think that's probably OK until we have a need to do those two things separately (e.g. register servers, but disable analytics). Right now we're expecting a single opt-out though.